server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsResponse.java

```diff
@@ -147,15 +145,14 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     @Override
-    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+    protected void addCustomXContentFields(XContentBuilder builder, Params params) throws IOException {
```

Comment:
this makes `IndicesStatsResponseTests#testInvalidLevel` fail. Could you please update the test so that it passes in a valid `XContentBuilder` rather than `null`?